src/Microsoft.Azure.NotificationHubs/ApnsCredential.cs

```diff
@@ -4,14 +4,15 @@
 // license information.
 //------------------------------------------------------------
 
+using System.Security.Cryptography;
+using Microsoft.Azure.NotificationHubs.Messaging;
+using System;
```

Comment:
Please clean up the imports by using sort/remove using statements so that they're in order.

Reply:
Done